pkg/kubeletconfig/consts.go

```diff
@@ -0,0 +1,5 @@
+package kubeletconfig
+
+const (
+	MinPodPidsLimit = 4096
```

Comment:
Same as the comment above, I wonder can we provide a way to fetch the limits from the API :thinking: